### DISCUSSION TITLE
On 1.4.3, but cannot access Open AI Assistants

### DISCUSSION THREAD

**Discussion by JohnA888** [FlowiseAI/Flowise#1331] [Q&A]
Hello, We updated to the lasted release, but do not see Open AI Assistant in the Agents list of nodes.  Is there something else to do?

**Comment by HenryHengZJ** [comment]
try to confirm if you can see your are on latest version: 
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/f941c354-0f9b-429f-8ca9-200933ffd29b)
